internal/cmd/kafka/command_cluster_create.go

```diff
@@ -142,32 +147,52 @@ func (c *clusterCommand) create(cmd *cobra.Command, args []string, prompt form.P
 		return err
 	}
 
-	encryptionKey, err := cmd.Flags().GetString("encryption-key")
+	var encryptionKey *string
```

Comment:
You can use `&` or `PtrString()` if you need this to be a pointer. Better not to over-complicate things.
```suggestion
	var encryptionKey string
```

Reply:
The cluster definition is eventually passed to cc-control-plane-kafka, which ensures that only byok or encryptionKey is set:
https://github.com/confluentinc/cc-control-plane-kafka/blob/1ac72463ee3b6aab0a5283910f5139bcc793b604/api/provisioning/v2/transport.go#L713
Problematically, `HasEncryptionKey` is only checking if the field is nil/not nil. Thus using a pointer here.

Reply:
Oh, it's not checking if the string is `""`? (as if the encryption key could be `""`? haha)